kmath-tensors/src/commonTest/kotlin/space/kscience/kmath/tensors/core/TestDoubleLinearOpsAlgebra.kt

```diff
@@ -0,0 +1,197 @@
+package space.kscience.kmath.tensors.core
+
+import space.kscience.kmath.tensors.core.algebras.DoubleLinearOpsTensorAlgebra
+import kotlin.math.abs
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class TestDoubleLinearOpsTensorAlgebra {
```

Comment:
Test classes are usually made `internal` to prevent problems if someone imports a test source set.